# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/debug/onEnterFrame-generator-01.js
# Commit: 73c33bf110d6
# Full Hash: 73c33bf110d6034266cbe615b9e018476a128d63
# Author: Logan Smyth <loganfsmyth@gmail.com>
# Date: 2019-12-09 09:50:39
# Regressor Bug: 1600204
# File Overlap Count: 2
# Description:
#   Bug 1600204 - Part 2: Remove Debugger.Frame.prototype.generator since it is already on script. r=jimb
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D55232
# ==============================================================================

diff -r d70ec1e79887 -r 73c33bf110d6 js/src/jit-test/tests/debug/onEnterFrame-generator-01.js
--- a/js/src/jit-test/tests/debug/onEnterFrame-generator-01.js	Fri Dec 06 02:29:18 2019 +0000
+++ b/js/src/jit-test/tests/debug/onEnterFrame-generator-01.js	Fri Dec 06 02:57:20 2019 +0000
@@ -48,7 +48,6 @@
     let a_expected = hits < 3 ? undefined : 1/2;
     assertEq(savedEnv.getVariable("a"), a_expected);
 
-    assertEq(frame.generator, true);
     assertEq(frame.onStack, true);
 
     let pc = frame.offset;
